deployment/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:8.15.0
+
+# Install VS Code's deps. These are the only two it seems we need.
+RUN apt-get update && apt-get install -y \
+	libxkbfile-dev \
+	libsecret-1-dev
+
+# Ensure latest yarn.
+RUN npm install -g yarn@1.13
+
+WORKDIR /src
+COPY . .
+
+# In the future, we can use https://github.com/yarnpkg/rfcs/pull/53 to make yarn use the node_modules
+# directly which should be fast as it is slow because it populates its own cache every time.
+RUN yarn && yarn task build:server:binary
+
+# Use golang base dev for go development tooling
```

Comment:
Not sure we should change the base image

Reply:
I'll leave it generic. My intent was to show how to have bootstrapped images. Having to install deps is not great, but doesn't make sense at this point in time.

Reply:
I like the idea of extending it.

We could include an example of something like:

```dockerfile
FROM codercom/code-server

RUN apt-get install mysql -y

...
```

Reply:
Any desire to move it to 1804. LTS for the base instead of 18.10 ?  Generally prefer to build docker images on things that have longer support.